internal/resources/bucket_schema.go

```diff
@@ -69,7 +64,6 @@ func BucketSchema() schema.Schema {
 			"time_to_live_in_seconds": schema.Int64Attribute{
 				Optional:            true,
 				Computed:            true,
-				Default:             int64default.StaticInt64(0),
```

Comment:
Can you add testing for these being removed as well?

Reply:
please see tests here https://jira.issues.couchbase.com/browse/AV-111247?focusedCommentId=1268023